### PR TITLE
[Fixed] Writing Unit Tests for Ensemble.py

### PR DESCRIPTION
**Pull request by ojhall94**
This PR supersedes #6  but contains the same core changes. 

I need to fix the Travis checks still.

### CODE REVIEW

File: tests/test_tesscorr.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    """Check that tesscorr.py returns the correct class"""
+    CorrClass = corrclass()
+    assert repr(CorrClass) == "<class 'corrections.ensemble.EnsembleCorrector'>"
+
+    CorrClass = corrclass('ensemble')
+    assert repr(CorrClass) == "<class 'corrections.ensemble.EnsembleCorrector'>"
+
+    CorrClass = corrclass('kasoc_filter')
+    assert repr(CorrClass) == "<class 'corrections.KASOCFilterCorrector.KASOCFilterCorrector'>"
+
```
Comment (rhandberg) [inline tests/test_tesscorr.py:30]: Add test for CBV as well

File: tests/test_ensemble.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+sector =1
+#----------------------------------------------------------------------
+def test_ensemble_basics():
+	"""Check that the Ensemblecorrector can be initiated at all"""
+	with corrections.EnsembleCorrector(INPUT_DIR, plot=True) as ec:
+		assert ec.__class__.__name__ == 'EnsembleCorrector', "Did not get the correct class name back"
+		assert ec.input_folder == INPUT_DIR, "Incorrect input folder"
+		assert ec.plot == True, "Plot parameter passed appropriately"
+
+@pytest.mark.skipif(os.environ["USER"] == 'travis',
```
Comment (rhandberg) [inline tests/test_ensemble.py]: Change condition to `os.environ.get('CI') == 'true' and os.environ.get('TRAVIS') == 'true'`

**Review by rhandberg — CHANGES_REQUESTED** [review]

File: tests/test_ensemble.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+	#Check contents
+	assert len(outlc.flux) == len(inlc.flux), "Input flux ix different length to output flux"
+	assert all(inlc.time == outlc.time), "Input time is nonidentical to output time"
+	assert all(outlc.flux != inlc.flux), "Input and output flux are identical."
+	assert len(outlc.flux) == len(outlc.time), "Check time and flux have same length"
+
+	#Check status
+	assert status == corrections.STATUS.OK, "STATUS was not set appropriately"
+
+@pytest.mark.skipif(os.environ["USER"] == 'travis',
```
Comment (rhandberg) [inline tests/test_ensemble.py]: Same thing should be changed here...

File: tests/test_ensemble.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+	#Check metadata
+	assert 'fmean' in outlc.meta, "Metadata is incomplete"
+	assert 'fstd' in outlc.meta, "Metadata is incomplete"
+	assert 'frange' in outlc.meta, "Metadata is incomplete"
+	assert 'drange' in outlc.meta, "Metadata is incomplete"
+	assert outlc.meta['task']['starid'] == inlc.meta['task']['starid'], "Metadata is incomplete"
+	assert outlc.meta['task'] == inlc.meta['task'], "Metadata is incomplete"
+
+@pytest.mark.skipif(os.environ["USER"] == 'travis',
```
Comment (rhandberg) [inline tests/test_ensemble.py]: ... and here

**Review by rhandberg — CHANGES_REQUESTED** [review]

File: tests/test_ensemble.py (hunk 4 above)
Comment (ojhall94) [reply tests/test_ensemble.py]: Oops... done!

**Comment by rhandberg** [comment]
Hi @ojhall94, have you pushed the last changes? From the "files changed" tab is still looks like there are some leftover `os.environ["USER"] == 'travis'`

**Comment by ojhall94** [comment]
Yep you're right, it looks like something funky has happened/is happening to my corrections repository, its not letting me push and telling me cross host redirect is not allowed... I'll try and fix this.

**Comment by ojhall94** [comment]
Okay, we're all good. That was weird!

**Review by rhandberg — APPROVED** [review]

**Pull request merged**